### DISCUSSION TITLE
Customize the map scale on Centering my location

### DISCUSSION THREAD

**Discussion by anastasiia936** [osmandapp/OsmAnd#11759] [Ideas]
Add the ability to set up the map scale that would not be automatically changed when I (re)center my location. 
It would be great if it would not need to adapt the scale after I (re)center my position. 
For example, if I previously set scale which was my preference to display the map, and I (re)center my location I have to change the map scale again (because by default app always (re)sets the scale to 200m). 
Would be especially useful for maritime and aviation profiles.

**Comment by nicknickny** [comment]
Exactly the same suggestion found on another thread posted by Sonora on 4 Nov 2015
https://github.com/osmandapp/OsmAnd/issues/1867

I personally find incorrect to change the map zoom to an arbitrary "neighborhood level" while scrolling the map and tapping the Location button. This could be corrected in two ways:
1. add a "Restore zoom when recenter map" option, or
2. the first tap centers the map, a second tap resets the zoom level.

**Comment by djh1961** [comment]
I also find the default zoom (scale) setting very inconvenient.  I use the app for motorcycling both on and off road where the speeds range from 10 - 70 mph.  Having only one default zoom setting really makes the app difficult to use.

Ideally, I would like the app to stay at the same zoom level when I select "center on own position" and tart navigating.
